features/graph/documentation/authentication.md

```diff
@@ -0,0 +1,103 @@
+# Graph Authentication
+
+Graph Authentication is the part of [Netlify Graph](README.md) that simplifies API authentication and token management. It removes the friction of re-writing authentication code to connect to third-party APIs.
+
+<!-- QUESTION: Are tokens still stored as env vars? -->
```

Comment:
@localden, I have this question in a few places. When I've authenticated in my testing, I haven't seen any new env vars. Are we handling tokens differently now?

Reply:
@KyleBlankRollins no, they are not - this is something that is now done through a Netlify-specific package. As in, you would not need to care about the Graph token.